components/blocks/tripleColumnImageBlock.tsx

```diff
@@ -0,0 +1,116 @@
+import Image from "next/image";
+import { Key } from "react";
+import type { Template } from "tinacms";
+import { isEmpty } from "../training/eventBooking";
+
+export type TripleColumnImageBlockProps = {
+  imageList: ImgCardProps[];
+};
+
+export type ImgCardProps = {
+  title: string;
+  subTitle?: string;
+  altText: string;
+  imageSrc: string;
+  height?: number | undefined;
+  width?: number | undefined;
+};
+
+export const TripleColumnImageBlock: React.FC<TripleColumnImageBlockProps> = ({
+  imageList,
+}) => {
+  return (
+    <div className="grid grid-cols-12 px-4 py-1">
+      {imageList?.map((imageblock, index: Key) => {
+        return <ImgCard {...imageblock} key={index} />;
+      })}
+    </div>
+  );
+};
+
+const ImgCard = ({
+  title,
+  subTitle,
+  imageSrc,
+  altText,
+  height,
+  width,
+}: ImgCardProps) => {
+  return (
+    <div className="col-span-4 py-4">
+      <h3 className="font-normal">{title}</h3>
+      <div className="h-4 text-xs font-normal text-slate-300">
+        {subTitle ?? ""}
+      </div>
```

Comment:
Can you use padding or margin here instead of a fixed height?

Reply:
I used fixed height to keep images on the same length regardless of its subTitle.